### DISCUSSION TITLE
Welcome to practices Discussions!

### DISCUSSION THREAD

**Discussion by maurolepore** [2DegreesInvesting/practices#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by maurolepore** [comment]
@MonikaFu, "Discussions" are a new feature of GitHub we now have access to. Do you think it would be useful? Maybe it could replace/complement the discussions we are now having on Slack. I sometimes wish those discussion were public.

**Reply by MonikaFu** [reply]
Yes, it looks indeed like a good place for some of our discussions!

**Comment by cjyetman** [comment]
I think "Discussions" would be great for public pkgs... to give user support... instead of StackOverflow, or instead of having GitHub issues flooded with support-style questions.
